services/docker-compose-ops.yml

```diff
@@ -111,6 +111,7 @@ services:
       - "4318:4318"  # OTLP HTTP receiver
     networks:
       - simcore_default
+      - interactive_services_subnet
```

Comment:
@YuryHrytsuk @mrnicegyu11 this also needs to be done in the OPS repository

Reply:
No it will not work. This `interactive_services_subnet ` one has tot be the one that is added. If I remember correctly dynamic-sidecars are not part of the `monitored` network

Reply:
As I remembered there is only one network added
 
<img width="550" alt="Screenshot 2024-11-13 at 09 03 40" src="https://github.com/user-attachments/assets/e1b4fd30-36d4-4310-8d60-d2ed87453162">

Reply:
Ok. I think this is a better approach from security perspective